portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.data;
+
+import java.time.Duration;
+
+/** Models a task. It can be used for both creation and import flow. */
+public final class Task {
+
+  private final String name;
```

Comment:
Similarly, please indicate which fields are optional and which fields can store null.

Consider using java.util.Optional for fields that can be optional.

Reply:
Done.